.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "workbench.colorCustomizations": {
+        "activityBar.background": "#073147",
+        "titleBar.activeBackground": "#094463",
+        "titleBar.activeForeground": "#F7FCFE"
+    }
+}
```

Comment:
Please add a new line here.

Reply:
A new line of what can u please guide :)

Reply:
A new line like this (just a simple **Enter** key click) :)
![image](https://user-images.githubusercontent.com/49204837/103506400-9fe17500-4e82-11eb-9c02-afc303047e1f.png)

Reply:
Ohhhh ok 😅

Reply:
You can view the pull request I try to fulfill change requests.